scss/_alert.scss

```diff
@@ -16,6 +16,11 @@
   > p + p {
     margin-top: 5px;
   }
+  
```

Comment:
Please remove the trailing whitespace on this line; it's making the Travis build fail.